### ISSUE TITLE
Got a crash. 

### ISSUE THREAD

**Issue by ThisJustInLocal** [bl4ckscor3/SuspiciousZombification#3]
Seems to be related to hordes? I have it set so zombies can attack and convert animals into their zombified counterparts and those counterparts convert other convertable mobs into their zombified forms, but it seems that the handler for your mod and mobs zombifying each other causes a crash. My best guess is a zombified pig tried to zombify another pig and both the handler for zombification for your mod and hordes activated at the same time and broke the game.

[crash-2024-07-27_15.33.56-server.txt](https://github.com/user-attachments/files/16401297/crash-2024-07-27_15.33.56-server.txt)

**Comment by ThisJustInLocal** [comment]
Can confirm. Tested it myself.

**Comment by bl4ckscor3** [comment]
As stated in the crash report: Have you reported this to the [Connector issue tracker](https://github.com/Sinytra/Connector/issues)?

**Comment by ThisJustInLocal** [comment]
> As stated in the crash report: Have you reported this to the [Connector issue tracker](https://github.com/Sinytra/Connector/issues)?

I do not believe this is a connector issue. I will test without connector and see if this crash still happens.

**Comment by ThisJustInLocal** [comment]
> As stated in the crash report: Have you reported this to the [Connector issue tracker](https://github.com/Sinytra/Connector/issues)?

Can confirm. Connector has nothing to do with it.
[crash-2024-07-27_18.52.36-server.txt](https://github.com/user-attachments/files/16401658/crash-2024-07-27_18.52.36-server.txt)

**Comment by ThisJustInLocal** [comment]
> > As stated in the crash report: Have you reported this to the [Connector issue tracker](https://github.com/Sinytra/Connector/issues)?
> 
> Can confirm. Connector has nothing to do with it. [crash-2024-07-27_18.52.36-server.txt](https://github.com/user-attachments/files/16401658/crash-2024-07-27_18.52.36-server.txt)

Double confirmed. Got it to crash twice even without forgified fabric.
[crash-2024-07-27_19.02.28-server.txt](https://github.com/user-attachments/files/16401671/crash-2024-07-27_19.02.28-server.txt)

**Comment by ThisJustInLocal** [comment]
Also, another bug issue I've noticed. Cats that get zombified happen to change their variant for some reason.

**Comment by bl4ckscor3** [comment]
This crash should now be fixed in v1.3.
